res/map/Spann_Teamplay.map

```diff
@@ -1,10 +0,0 @@
-  SE  SE  SE  SE  SE  SE  SE  SE  SE  SE  SE  SE  SE  SE  SE
-  SE  SE  GR  FR  RD  RD  BR  RD  RD  GR  FR 0FC  GR 0FC  SE
-  SE  GR  GR  FR  RD  FR  SE  GR  RD  CT  GR  GR 3HQ 3FC  SE
-  SE  CT  CT  RD  RD  SE  SE  SE  RD  GR  MT  GR  RD 3FC  SE
-  SE  GR  GR  RD  GR  SE  SE  SE  RD 0HQ  MT  CT  RD  GR  SE
-  SE 2FC  RD  RD 1HQ  SE  SE  FR  RD  RD  RD  RD  RD  CT  SE
-  SE 2FC 2HQ  GR  GR  GR  GR  FR  FR  GR  SE  SE  BR  SE  SE
-  SE  SE 1FC 1FC  GR  GR  GR  MT  GR  GR  SE  SE  GR  GR  SE
-  SE  RF  SE  SE  MT  CT  MT  FR  FC  GR  BR  GR  CT  CT  SE
-  SE  SE  SE  SE  SE  SE  SE  SE  SE  SE  SE  SE  SE  SE  SE
```

Comment:
You clearly just have no taste.
It's like a fine wine, I tell you!

Reply:
I don't appreciate fine wines, either. Or escargot, if we want to add in that comparison.